lib/wallet/txdb.js

```diff
@@ -958,6 +958,14 @@ class TXDB {
         state.coin(path, -1);
         state.unconfirmed(path, -coin.value);
 
+        // FINALIZE is a special case: locked coins _leave_ the wallet.
```

Comment:
I realized that this could be simplified even more:

```js
if (tx.output(i) && tx.covenant(i).isFinalize())
```

https://github.com/handshake-org/hsd/blob/22be7faa37b1a948924883394b28747c348d8014/lib/primitives/covenant.js#L324

Reply:
dude yes! thank you, applying...